loopout_test.go

```diff
@@ -99,3 +101,158 @@ func TestLateHtlcPublish(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// TestCustomSweepConfTarget ensures we are able to sweep a Loop Out HTLC with a
+// custom confirmation target.
+func TestCustomSweepConfTarget(t *testing.T) {
+	defer test.Guard(t)()
+
+	lnd := test.NewMockLnd()
+	ctx := test.NewContext(t, lnd)
+
+	// Use the highest sweep confirmation target before we attempt to use
+	// the default.
+	testRequest.SweepConfTarget = testLoopOutOnChainCltvDelta -
+		DefaultSweepConfTargetDelta - 1
+
+	// Set up custom fee estimates such that the lower confirmation target
+	// yields a much higher fee rate.
+	ctx.Lnd.SetFeeEstimate(testRequest.SweepConfTarget, 250)
+	ctx.Lnd.SetFeeEstimate(DefaultSweepConfTarget, 10000)
+
+	cfg := &swapConfig{
+		lnd:    &lnd.LndServices,
+		store:  newStoreMock(t),
+		server: newServerMock(),
+	}
+	swap, err := newLoopOutSwap(
+		context.Background(), cfg, ctx.Lnd.Height, testRequest,
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Set up the required dependencies to execute the swap.
+	sweeper := &sweep.Sweeper{Lnd: &lnd.LndServices}
+	blockEpochChan := make(chan interface{})
+	statusChan := make(chan SwapInfo)
+	expiryChan := make(chan time.Time)
```

Comment:
Creating test context like `loopInTestContext` for loop in would be good to reduce code duplication and maintenance cost in the future. Maybe it is too much for a 1-character PR though. Otherwise you could add a TODO in both of these tests so that the next dev might do it.

Reply:
Do you want to add the TODO?

Reply:
Done.